account_chart_report/wizard/account_report_chart_of_account.py

```diff
@@ -20,24 +20,22 @@
 #
 ##############################################################################
 
-from openerp.osv import fields, orm
+from openerp import fields, models, api
 
 
-class ChartOfAccountsReport(orm.TransientModel):
+class ChartOfAccountsReport(models.TransientModel):
     _name = 'account.print.chart.accounts.report'
     _description = 'Chart of accounts Report'
 
-    domain_char_account = [('parent_id', '=', False)]
-    _columns = {
-        'chart_account_id': fields.many2one('account.account',
-                                            'Chart of Accounts',
-                                            help='Select Charts of Accounts',
-                                            required=True,
-                                            domain=domain_char_account),
-    }
+    chart_account_id = fields.Many2one('account.account', 
+                                       'Chart of Accounts',
+                                       help='Select Charts of Accounts',
+                                       required=True,
+                                       domain=([('parent_id', '=', False)]))
 
-    def print_report(self, cr, uid, ids, data, context=None):
-        res = self.read(cr, uid, ids, context=context)[0]
+    @api.multi
+    def print_report(self, data):
+        res = self.read()[0]
```

Comment:
instead of using read, you can directly get the value.
(I assume `self` can only be a single ChartOfAccountsReport)


    self.chart_account_id.id